config/languages/sp.ini

```diff
@@ -0,0 +1,3 @@
+[Common]
```

Comment:
Google translate? Or do you know spanish?

Reply:
I know some, but some of the phrases were translated by others.